new-client/src/components/Window.js

```diff
@@ -516,6 +516,7 @@ class Window extends React.PureComponent {
             {features && features.length > 0 ? (
               <FeatureInfoContainer
                 features={this.props.features}
+                options={this.props.options}
```

Comment:
Not necessary but we could destructure props instead

Reply:
`{options}={{this.props}}`? Why?

Reply:
Sorry i meant all the props, but not necessary as i said

`const  {options, features, onDisplay, globalObserver}` = this.props;